community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_1/TransactionBoundPlanContext.scala

```diff
@@ -75,7 +73,7 @@ class TransactionBoundPlanContext(tc: TransactionalContextWrapper, logger: Inter
   private def evalOrNone[T](f: => Option[T]): Option[T] =
     try { f } catch { case _: SchemaKernelException => None }
 
-  private def getOnlineIndex(descriptor: NewIndexDescriptor): Option[IndexDescriptor] =
+  private def getOnlineIndex(descriptor: NewIndexDescriptor): Option[CypherIndexDescriptor] =
```

Comment:
Could have replaced 'New' with 'Kernel' and then not needed to make all the 'Cypher' prefixes.